helm/cluster-api-provider-aws/values.yaml

```diff
@@ -1,5 +1,5 @@
 name: cluster-api-provider-aws
-tag: 093033f3c6f8a0660fdd913b94435133e62991b1
+tag: b11e209398ed27dcd47c12b1f8385f4dc73d52e1
```

Comment:
The way we currently push images in our fork (using upstream makefile), images are still named with the commit. I would prefer to have the version tag here in the future.

Reply:
The Version tag does not appear on the image that is pushed to the registry. It's because we do not build these images how we usually do in circleCI, but using the Makefile that was already there.We have to make changes to either CircleCI or the Makefile there.

Reply:
Thanks for explaining 🙂 . We can followup to fix this later 👍🏻 .

Reply:
Yeah let's get a release first :D